docs/managing-program-users.md

```diff
@@ -3,4 +3,14 @@ id: managing-program-users
 title: Managing Program Users
 ---
 
-Placeholder
+If you are a Program Administrator, then you will have the ability to invite collaborators to access your programs data.
+
+## How to Add Users
+1. Navigate to the **Submission** area in the top menu.
+1. Click **Manage Programs** in the .
+1. Above the users table, click the *Add Users* button.
+1. Fill in the First Name, Last Name, Email, and role of the person you would like to collaborate on your ARGO program.
```

Comment:
...of the person you would like to add to your ARGO program.
(just because "collaborator" is a role name)

Reply:
re ICGC- too many letters everywhere all the time :/ maybe we dont need to say it everywhere instead...

Reply:
branding is important too - not just less words :)
We may need to see what they say on the website as it should be consistent with how they refer to the program across the board.